CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing
+
+Thanks a lot for your interest in contributing to **editorconfig-checker**!
+
+## Types of contributions
+
+Contributions to **editorconfig-checker** _include_, but is _not limited to_:
+
+- Report a bug.
+- Suggest a new feature idea.
+- Correct spelling errors, improvements, or additions to documentation files (README, CONTRIBUTING...).
+- Improve structure/format/performance/refactor/tests of the code.
+
+All contributions are welcome from anyone willing to work in good faith with other contributors and the community. No contribution is too small and all contributions are valued.
+
+## Open Development
+
+All work on **editorconfig-checker** happens directly on [GitHub](https://github.com/editorconfig-checker/editorconfig-checker). Both core team members and external contributors send pull requests, which go through the same review process.
+
+## Pull Requests
+
+Pull Requests are the way concrete changes are made to the code and the documentation.
+
+**Please first discuss** the change you wish to make via [issue](https://github.com/editorconfig-checker/editorconfig-checker/issues) before making a change. It might save you time.
+
+If you're adding new features to **editorconfig-checker**, please include tests.
+
+### Review process and requirements to merge
+
+To ensure everyone in the community has the opportunity to share their feedback, ideas, and suggestions, a Pull Request must go through a review process, and must meet the following requirements to be merged:
+
+- **Approval:** At least **2 core team members** must approve the Pull Request.
+- **Open Duration:** The Pull Request must be **open for at least 72 hours**.
+- **CI checks:** The Pull Request must **pass all CI checks**, including tests and build.
+
+**Note:** If a critical bug fix is required, and the Pull Request was approved by at least **3 core team members**, the Pull Request **can be merged before the 72 hours** open duration.
+
+Pull requests are merged using the `Squash and Merge` option to keep the commit history clean, and promote clear separation of changes in separated pull requests.
+
+This collaborative process ensures that all perspectives are considered, and helps improve the project quality as a whole.
+
+## Commits and Releases
+
+The commit message guidelines adheres to [Conventional Commits](https://www.conventionalcommits.org/) and [Semantic Versioning](https://semver.org/) for releases.
```

Comment:
I'm fine with this, but then you should add something to CI to check all commits respect the policy.

It also requires to configure dependabot to use this pattern when opening PRs

Reply:
There is already `wagoid/commitlint-github-action` to check that commits respect the policy in CI.
And dependabot is already configured, to respect this commit policy. :smile:

It was however not documented.

Reply:
CI does that already - I learned that the hard way ;)